src/main.js

```diff
@@ -12,28 +18,23 @@ var currentCover;
 
 // Add your event listeners here 👇
 
-
 // Create your event handlers and other functions here 👇
 
 
 // We've provided one function to get you started
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
-}
-
-var newCover = covers[getRandomIndex(covers)];
-var newTitle = titles[getRandomIndex(titles)];
-var newDescrip1 = descriptors[getRandomIndex(descriptors)];
-var newDescrip2 = descriptors[getRandomIndex(descriptors)];
-
-// decide where this goes
-if (newDescrip1 === newDescrip2) {
-  newDescrip2 = descriptors[getRandomIndex(descriptors)];
-}
-
-currentCover = new Cover(newCover, newTitle, newDescrip1, newDescrip2);
-
-coverImage.src = currentCover.cover;
-coverTitle.innerText = currentCover.title;
-descriptor1.innerText = currentCover.tagline1;
-descriptor2.innerText = currentCover.tagline2;
+};
+function showRdmCover() {
```

Comment:
I am glad we were able to refactor that function and use it for both iteration 0 and 1.